src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -160,6 +160,7 @@ private BoundAttribute BindAttributeCore(AttributeSyntax node, NamedTypeSymbol a
                 {
                     if (Compilation.LanguageVersion < MessageID.IDS_FeatureStaticNullChecking.RequiredVersion())
                     {
+                        // PROTOTYPE(NullableReferenceTypes): Stop injecting NonNullTypesAttribute and get rid of this error
```

Comment:
>PROTOTYPE [](start = 27, length = 9)

Please use a github issue to track this instead

Reply:
> Please use a github issue to track this instead

I am targeting a feature branch. Once the PR is merged, I will open issues for comments that we do not plan to address before merging up. Some of the comments should be addressed when we add "real" directive. 

---
In reply to: [223879622](https://github.com/dotnet/roslyn/pull/30387#discussion_r223879622) [](ancestors = 223879622)